### PR TITLE
[changelog] Add missing entry regarding api config reload

### PR DESCRIPTION
**Pull request by morucci**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ All notable changes to this project will be documented in this file.
 ### Removed
 ### Fixed
 
+- [api] workspaces' crawler entities not created/refreshed at config changes (#739).
```
Comment (TristanCacqueray) [inline CHANGELOG.md:23]:
```suggestion
- [api] workspaces' crawler entities not created/refreshed at config changes [#739](https://github.com/change-metrics/monocle/pull/739).
```

**Comment by TristanCacqueray** [comment]
It seems like we could automate that part using https://github.com/haskell/haskell-language-server/blob/master/GenChangelogs.hs

**Comment by morucci** [comment]
@TristanCacqueray I've created https://github.com/change-metrics/monocle/issues/761 to move forward with your proposal.

**Comment by morucci** [comment]
Would you like to give your approval on that change in order we merge this entry in the changelog ?

**Review by TristanCacqueray — APPROVED** [review]

**Pull request merged**